tuning/config/acceleration_configs/acceleration_framework_config.py

```diff
@@ -89,6 +91,17 @@ class AccelerationFrameworkConfig:
         ),
     ] = None
 
+    fast_moe: Annotated[
+        FastMoe,
+        ConfigAnnotation(
+            path="training.moe",
+            key="scattermoe",
+            standalone=True,
+            experimental=False,
```

Comment:
`standalone=True` for now. but in the upcoming issue https://github.com/foundation-model-stack/fms-acceleration/issues/103 im thinking how to avoid using the `model_loader`. It is really only used just to intercept `model_name_or_path`.

`experimental` should be set to `True`